### ISSUE TITLE
Only pasting half of a model

### ISSUE THREAD

**Issue by HelenaKitty** [cuberite/WorldEdit#120]
For some reason, WorldEdit is only pasting half of my model

Original model
![2017-02-27_16 26 47](https://cloud.githubusercontent.com/assets/518378/23369591/cda7aec4-fd09-11e6-9095-6371662f4bf7.png)

Pasted model
![2017-02-27_16 27 03](https://cloud.githubusercontent.com/assets/518378/23369603/d97879f4-fd09-11e6-99f5-2a1cf66c94ba.png)

**Comment by NiLSPACE** [comment]
Is it reproducable? If so could you use copy it, use `//schematic save <name here>` and send the schematic?

Strange that it is only happening with a small piece inside of the structure. @madmaxoft any idea what's happening?

**Comment by HelenaKitty** [comment]
Yes, already tried pasting it twice and get the same outcome, I am going to prepare a schematic.

**Comment by HelenaKitty** [comment]
I have uploaded the schematic which you have requested.

[greenhouse.zip](https://github.com/cuberite/WorldEdit/files/804336/greenhouse.zip)

**Comment by NiLSPACE** [comment]
Thank you, I'll take a look at it.

**Comment by HelenaKitty** [comment]
No problem, and if there is anything else you need regarding testing then just gimme a shout, I will see what I can do, or know to do.

**Comment by NiLSPACE** [comment]
Strangely it seems chunk-specific, which sadly means this is most likely a problem in Cuberite. @madmaxoft did the `cBlockArea` write code change lately?

![2017-02-27_17 48 13](https://cloud.githubusercontent.com/assets/1160867/23370435/f9d63abe-fd14-11e6-83c3-8e5f88ec25a1.png)
![2017-02-27_17 48 20](https://cloud.githubusercontent.com/assets/1160867/23370436/f9d65a3a-fd14-11e6-8c54-d4132bb57682.png)

**Comment by HelenaKitty** [comment]
But, Cuberite development is extremely active, I trust that this will be sorted.

**Comment by HelenaKitty** [comment]
I have just found out that the model shows in full after a server restart.

**Comment by NiLSPACE** [comment]
Good catch! That means it's most likely an issue with the new `1.11` protocol.

**Comment by HelenaKitty** [comment]
Which is the most stable protocol from the best of your knowledge, I don't mind about unknown bugs, however I am going to my server to the most stable protocol version.

I'm thinking 1.10.x  for now.

**Comment by NiLSPACE** [comment]
`1.11` is the latest protocol that was introduced, so it's most likely not the stablest. If you want a stable client you'll have a bigger chance if you'd use an older client like `1.8`, which is the oldest version we support. Though `1.10` will most likely work really good as well though.

I'd open an issue on Cuberite for this though, WorldEdit can't fix this I'm afraid.

**Comment by HelenaKitty** [comment]
In regards to 
> Good catch! That means it's most likely an issue with the new 1.11 protocol.

I get the same issue on 1.10.

**Comment by HelenaKitty** [comment]
After doing further testing, I have pasted around some other schematics, and they pasted fine with 1.9.
